### PR TITLE
Plot Author Publication Data

### PR DESCRIPTION
**Pull request by asl3**
Graph counts of how many papers of our master_papers.csv dataset each author wrote

Screen Shot 2023-11-20 at 9 26 41 PM

Screen Shot 2023-11-20 at 10 00 34 PM

Frequency Table:

```
    Publication Count  Frequency
0                   1       5536
1                   2        658
2                   3        178
3                   4         67
4                   5         35
5                   6         17
7                   7          4
6                   8          6
11                 10          1
10                 11          1
8                  13          2
9                  20          1
```

### CODE REVIEW

**Comment by trigaten** [comment]
Please black the code

File: scripts/visualize_authors.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Read the CSV file into a DataFrame
+arxiv_papers_df = pd.read_csv(file_path)
+
+# Split authors into a list and explode them to have one row per author
+authors = arxiv_papers_df['authors'].str.replace("'", "").str.replace("]", "").str.replace("[", "").str.split(',').explode()
+
+# Count the occurrences of each author
+author_counts = authors.value_counts()
+
+# make a vertical bar chart displaying top 20 authors publicaiton counts
```
Comment (trigaten) [inline scripts/visualize_authors.py:17]: typo

File: scripts/visualize_authors.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+plt.yscale('log')  # Logarithmic scale
+
+plt.xticks(frequency_table["Publication Count"])
+
+plt.xlabel('Publication Count')
+plt.ylabel('Frequency (log scale)')
+plt.title('Publication Count vs Frequency (Log Scale)')
+plt.grid(True, which="both", ls="--", linewidth=0.5)
+
+plt.show()
```
Comment (trigaten) [inline scripts/visualize_authors.py:50]: If I run this script, where do the graphs go? Are they saved to disc?

Comment (asl3) [reply scripts/visualize_authors.py:50]: I guessed we don’t want to save the graphs to the repo, so the plt.show() just makes the graph displays pop up, and the script returns when the user closes out the displays. Do you think we should save graphs to disc?

Comment (trigaten) [reply scripts/visualize_authors.py:62]: Oh cool, no this is fine. I just didn't know what behavior occurred here.

**Pull request merged**